pkg/api/meta/meta.go

```diff
@@ -28,6 +29,40 @@ import (
 	"github.com/golang/glog"
 )
 
+func ListAccessor(obj interface{}) (List, error) {
+	if oi, ok := obj.(ListMetaAccessor); ok {
```

Comment:
I'm generally in favor of short variable names within small functions, but I can't figure out what these letters stand for, so maybe write it out?